src/GraphQL.DataLoader.Tests/GraphQL.DataLoader.Tests.csproj

```diff
@@ -7,12 +7,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-      <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.0.0" />
-      <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
+      <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.1.1" />
```

Comment:
Changing this package version to `2.1` should be done at the same time as when the project is updated to `2.1`.  It may need to multi-target `2.0` as well as `2.1`.